### DISCUSSION TITLE
Error al usar el componente EmptyMessage de patterns

### DISCUSSION THREAD

**Discussion by hrchioest** [TiendaNube/nimbus-design-system#158] [Q&A] [answered]
Buenas tardes, reporto este problema que me surgió al usar el componente EmptyMessage de patterns, descargue la versión: "@nimbus-ds/patterns": "1.3.0", y me mostraba este error en el navegador:

![Captura de Pantalla 2023-05-11 a la(s) 12 55 24](https://github.com/TiendaNube/nimbus-design-system/assets/53129859/2157061c-91b6-4d39-8890-54fd32ac3d6f)

Obs: También se estuvo reportando otro caso en discussion de patterns: https://github.com/TiendaNube/nimbus-patterns/discussions

**Comment by juniorconquista** [comment, marked as answer]
@hrchioest nesse caso basta apenas as versõs dos seguintes pacotes: `"@nimbus-ds/components", "@nimbus-ds/patterns" e "@nimbus-ds/styles"`.
